atom/browser/api/atom_api_app.cc

```diff
@@ -331,10 +330,6 @@ struct Converter<Browser::LoginItemSettings> {
                                    Browser::LoginItemSettings val) {
     mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
     dict.Set("openAtLogin", val.open_at_login);
-    dict.Set("openAsHidden", val.open_as_hidden);
-    dict.Set("restoreState", val.restore_state);
-    dict.Set("wasOpenedAtLogin", val.opened_at_login);
-    dict.Set("wasOpenedAsHidden", val.opened_as_hidden);
```

Comment:
There seems to be a workaround for this. Should be documented: https://stackoverflow.com/questions/33181336/os-x-how-to-detect-when-app-got-launched-in-hidden-state-at-login?noredirect=1&lq=1